system/ee/ExpressionEngine/Controller/Settings/MenuManager.php

```diff
@@ -218,6 +230,19 @@ private function form($set_id = null)
 
             if ($result->isValid()) {
                 $set->save();
+
+                if (defined('CLONING_MODE') && CLONING_MODE === true && $kids->count() == 0) {
+                    $originalKids = ee('Model')->get('MenuItem')->filter('set_id', $originalSetId)->all();
```

Comment:
this part drops dropdown kids because their set id is set to null and their parent id is what needs to be referenced